Open-Rival/Open-Rival.vcxproj

```diff
@@ -77,14 +78,16 @@
     <ClInclude Include="include\TileRenderer.h" />
     <ClInclude Include="include\TimerUtils.h" />
     <ClInclude Include="include\Unit.h" />
+    <ClInclude Include="include\UnitAnimation.h" />
+    <ClInclude Include="include\UnitAnimationLookup.h" />
     <ClInclude Include="include\UnitRenderer.h" />
     <ClInclude Include="include\Window.h" />
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{AE6B3FDA-A900-4013-82C7-110DA7895525}</ProjectGuid>
     <RootNamespace>OpenRival</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
Fur future reference, `git commit -p [file]` allows you to commit specific parts of a file. It shows you each "hunk" that you have changed within the file, and you can choose whether or not to stage it.